### PR TITLE
Add cg-ui dashboard to clients.yml

### PR DESCRIPTION
**Pull request by hursey013**
## Changes proposed in this pull request:
- Add new UAA client in order to deploy and evaluate `paas-admin` as per [documentation](https://github.com/alphagov/paas-admin?tab=readme-ov-file#prerequisites)

## security considerations
Clients.yml is currently deployed on every environment so this new client will be available in production as well.

### CODE REVIEW

File: bosh/opsfiles/clients.yml [hunk 1]
```diff
@@ -144,6 +144,17 @@
     redirect-uri: https://dashboard.((system_domain))/pp/v1/auth/sso_login_callback
     secret: ((stratos-client-secret))
 
+- type: replace
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaa/clients/cg-ui?
+  value:
+    override: true
+    authorized-grant-types: authorization_code,client_credentials,refresh_token
+    autoapprove: true
+    secret: ((cg-ui-client-secret))
```
Comment (markdboyd) [inline bosh/opsfiles/clients.yml:153]: In order for a secret value to be generated for your UAA client, you also need to copy these lines in this file and update the name to `cg-ui-client-secret`

https://github.com/cloud-gov/cg-deploy-cf/blob/64e0df22ea4c3cfe2c6862792fbb18b6712abddc/bosh/opsfiles/use-master-bosh-ca.yml#L243-L247

Comment (hursey013) [reply bosh/opsfiles/clients.yml:153]: Thanks, I've added it there as well.

File: bosh/opsfiles/clients.yml [hunk 2]
```diff
@@ -144,6 +144,17 @@
...
+    scope: cloud_controller.read,cloud_controller.admin_read_only,cloud_controller.global_auditor,cloud_controller.write,scim.me,openid,profile,uaa.user,cloud_controller.admincloud_controller.read,cloud_controller.admin_read_only,cloud_controller.global_auditor,cloud_controller.write,scim.me,openid,profile,uaa.user,cloud_controller.admin
```
Comment (markdboyd) [inline bosh/opsfiles/clients.yml:154]: Looks like there are some duplicate or invalid scopes in here. The correct/minimum set looks like

```
['cloud_controller.read', 'cloud_controller.admin_read_only', 'cloud_controller.global_auditor', 'cloud_controller.write', 'scim.me', 'openid', 'profile', 'uaa.user', 'cloud_controller.admin']
```

Comment (hursey013) [reply bosh/opsfiles/clients.yml]: Argh, you're right, looks like the paas-admin docs has it copied twice. Updated.

File: bosh/opsfiles/clients.yml [hunk 3]
```diff
@@ -144,6 +144,17 @@
...
     secret: ((stratos-client-secret))
 
+- type: replace
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaa/clients/cg-ui?
+  value:
+    override: true
+    authorized-grant-types: authorization_code,client_credentials,refresh_token
+    autoapprove: true
+    secret: ((cg-ui-client-secret))
+    scope: cloud_controller.read,cloud_controller.admin_read_only,cloud_controller.global_auditor,cloud_controller.write,scim.me,openid,profile,uaa.user,cloud_controller.admin
```
Comment (markdboyd) [inline bosh/opsfiles/clients.yml:154]:
```suggestion
    scope: scim.me,openid,profile,uaa.user,cloud_controller.admin
```

If you have `cloud_controller.admin` you have everything else, so we can simplify this a lot

https://docs.cloudfoundry.org/concepts/architecture/uaa.html#cc-scopes

**Review by markdboyd — APPROVED** [review]

**Pull request merged**